src/test/mochitest/head.js

```diff
@@ -223,6 +224,10 @@ function waitForSelectedSource(dbg, sourceId) {
   });
 }
 
+function isSourceLoaded(source) {
```

Comment:
No special role. I just thought for test files they had to be defined separately!